notifications_utils/recipient_validation/phone_number.py

```diff
@@ -16,6 +16,18 @@
 
 UK_PREFIX = "44"
 
+ALLOW_LIST = [
+    phonenumbers.PhoneNumberType.FIXED_LINE,
+    phonenumbers.PhoneNumberType.MOBILE,
+    phonenumbers.PhoneNumberType.FIXED_LINE_OR_MOBILE,
```

Comment:
As far as I can tell, they mean different things.

So each number type in that list basically evaluates to an int, so when you do comparisons like `phonenumbers.number_type(phone_number) == phonenumebrs.PhoneNumberType.MOBILE`.

That expression will evaluate to something like `2 == 2` --> True.

So because each phone number will have a single unique type, we need to check for everything.

Reply:
@rparke will add a comment explaining the ambiguous case.